src/main/java/fr/jmmc/oiexplorer/core/gui/SliderEditor.java

```diff
@@ -0,0 +1,202 @@
+        });
+        gridBagConstraints = new java.awt.GridBagConstraints();
+        gridBagConstraints.gridx = 1;
+        gridBagConstraints.gridy = 2;
+        getContentPane().add(jConfirmButton, gridBagConstraints);
+
+        jCancelButton.setText("Cancel");
+        jCancelButton.addActionListener(new java.awt.event.ActionListener() {
+            public void actionPerformed(java.awt.event.ActionEvent evt) {
+                jCancelButtonActionPerformed(evt);
+            }
+        });
+        gridBagConstraints = new java.awt.GridBagConstraints();
+        gridBagConstraints.gridx = 2;
+        gridBagConstraints.gridy = 2;
+        getContentPane().add(jCancelButton, gridBagConstraints);
+
+        jRangeSliderPanel.setLayout(new java.awt.GridBagLayout());
+        gridBagConstraints = new java.awt.GridBagConstraints();
+        gridBagConstraints.gridx = 1;
+        gridBagConstraints.gridy = 0;
+        gridBagConstraints.gridwidth = 2;
+        getContentPane().add(jRangeSliderPanel, gridBagConstraints);
+
+        jSpinnerMinimum.addChangeListener(new javax.swing.event.ChangeListener() {
+            public void stateChanged(javax.swing.event.ChangeEvent evt) {
+                jSpinnerMinimumStateChanged(evt);
+            }
+        });
+        gridBagConstraints = new java.awt.GridBagConstraints();
+        gridBagConstraints.gridx = 1;
+        gridBagConstraints.gridy = 1;
+        gridBagConstraints.fill = java.awt.GridBagConstraints.BOTH;
+        getContentPane().add(jSpinnerMinimum, gridBagConstraints);
+
+        jSpinnerMaximum.addChangeListener(new javax.swing.event.ChangeListener() {
+            public void stateChanged(javax.swing.event.ChangeEvent evt) {
+                jSpinnerMaximumStateChanged(evt);
+            }
+        });
+        gridBagConstraints = new java.awt.GridBagConstraints();
+        gridBagConstraints.gridx = 2;
+        gridBagConstraints.gridy = 1;
+        gridBagConstraints.fill = java.awt.GridBagConstraints.BOTH;
+        getContentPane().add(jSpinnerMaximum, gridBagConstraints);
+
+        pack();
+    }// </editor-fold>//GEN-END:initComponents
+
+    private void postInit() {
+        SpinnerNumberModel model = new SpinnerNumberModel(1, 1, this.imagesCount, 1);
+        this.jSpinnerMinimum.setModel(model);
+        this.jSpinnerMinimum.setValue(1);
+        model = new SpinnerNumberModel(1, 1, this.imagesCount, 1);
+        this.jSpinnerMaximum.setModel(model);
+        this.jSpinnerMaximum.setValue(this.imagesCount);
+    }
+
+    private void jCancelButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_jCancelButtonActionPerformed
+        this.setVisible(false);
+    }//GEN-LAST:event_jCancelButtonActionPerformed
+
+    private void jConfirmButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_jConfirmButtonActionPerformed
+        this.sliderPanel.setRange((Integer) jSpinnerMinimum.getValue(), (Integer) jSpinnerMaximum.getValue());
```

Comment:
Please add null check as sliderPanel can be null:
if (this.sliderPanel != null) {
... 
}